src/components/operations/operation-details/ko/runtime/operation-console.ts

```diff
@@ -194,7 +194,7 @@ export class OperationConsole {
         this.responseStatusText(null);
         this.responseBody(null);
 
-        const operation = await this.apiService.getOperation(selectedOperation.id);
+        const operation = await this.apiService.getOperation(`apis/${selectedApi.id}/operations/${selectedOperation.id}`);
```

Comment:
@harunrst any concerns on the suggested approach?

Reply:
no concern at all, but it might break the integrity of the codebase. Before this, ids were paths instead of an id. example id for operation i.e: 
old id: apis/123/operations/423
new id: 423
this approach is from ARM. So I replaced it with a single string when I was doing it. But also sending 2 parameters and concatenating them inside the function is also cool, actually more readable. However, all services are accepting id like this now because of ARM approach we had before.

Reply:
This one is updated. Thanks for the discussion!